third_party/libevent/METADATA

```diff
@@ -6,13 +6,13 @@ description:
 third_party {
   url {
     type: GIT
-    value: "https://github.com/libevent/libevent.git"
+    value: "https://chromium.googlesource.com/chromium/src"
   }
-  version: "3e9aad72214519ac2ad6f5f023af48136d0dd50d"
+  version: "70.0.3538.124"
```

Comment:
Any chance of adding a separate comment line with the semantic version of the upstream package ? e.g.
```
# Version: x.y.z
```

I know it's not machine readable, but it's better than nothing.

Reply:
Let's do this by reference to avoid a competing “source of truth” here.